### PR TITLE
Update minimap2 v.2.20

### PR DESCRIPTION
**Pull request by gallardoalba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

This PR updates minimap2 to version 2.20.

### CODE REVIEW

**Review by bgruening — COMMENTED** [review]
@gallardoalba nice. Can you please make sure that changes https://github.com/lh3/minimap2/releases/tag/v2.19 are included here?

**Comment by gallardoalba** [comment]
> @gallardoalba nice. Can you please make sure that changes https://github.com/lh3/minimap2/releases/tag/v2.19 are included here?

I also added the -U parameter; I think that now everything has been included, although some parameters hadn't included in the previous tool versions (e.g. --hard-mask-level); Should I include them?

**Comment by bgruening** [comment]
Just put it somewhere on the todo list. Its not super important afaik. Its more important that we update potential default settings etc... but if you have time. Non one it stopping you :)

**Review by bgruening — APPROVED** [review]

File: tools/minimap2/minimap2.xml [hunk 1]
```diff
@@ -1,20 +1,12 @@
...
-        </xml>
+        <import>macros.xml</import>
     </macros>
-    <requirements>
-        <requirement type="package" version="@TOOL_VERSION@">minimap2</requirement>
-        <requirement type="package" version="1.9">samtools</requirement>
-    </requirements>
+    <expand macro="edam_ontology"/>
+    <expand macro="requirements"/>
+    <expand macro="pe_anaylsis_fixed_selector"/>
```
Comment (wm75) [inline tools/minimap2/minimap2.xml:9]: Uha - this is just wrong in this place. The macro gets expanded further down in the inputs section.

Comment (gallardoalba) [reply tools/minimap2/minimap2.xml:9]: I'll fix that; I don't understand why the tests didn't fail.

Comment (wm75) [reply tools/minimap2/minimap2.xml:9]: Don't worry. Should get fixed in #3736

**Pull request merged**